configure/os/CONFIG.Common.RTEMS-qoriq_e500

```diff
@@ -0,0 +1,48 @@
+#
+# CONFIG.Common.RTEMS-qoriq_e500
+# Author: Heinz Junkes <junkes@fhi-berlin.mpg.de> 
+#
+# All RTEMS targets use the same Makefile fragment
+#
+EXE = .elf
+RTEMS_BSP = qoriq_e500
+RTEMS_TARGET_CPU = powerpc
+GNU_TARGET = powerpc-rtems
+# optimization trouble in postfix.c
+ARCH_DEP_CFLAGS += -DRTEMS_HAS_ALTIVEC
+#will use bootp
+#ARCH_DEP_CFLAGS += -DMY_DO_BOOTP=NULL
+
+#ARCH_DEP_CFLAGS += -DRTEMS_NETWORK_CONFIG_MBUF_SPACE=2048
+#ARCH_DEP_CFLAGS += -DRTEMS_NETWORK_CONFIG_CLUSTER_SPACE=5120
+
+#netbsdlib
+#ARCH_DEP_CFLAGS += -I$(RTEMS_BASE)/powerpc-rtems5/qoriq_e500/lib/include
+
+#OP_SYS_LDLIBS += -lbspExt #does not use posix stuff ... want to ignore
```

Comment:
Does this imply that the VME bus support isn't working for the mvme2500 target?

Reply:
I heard that from @hjunkes a few weeks ago. The APS will need it though, so unless Heinz gets to it first that's going to be another task for @brendanchandler after this port has been merged.

Reply:
I vote for addressing this in a separate pull request.

Reply:
> I vote for addressing this in a separate pull request.

To clarify, I want to deal with VME bus support in a separate pull request, as Andrew suggested.  Not suggesting this change be put in a separate pull request.  Sorry for the confusion there.